### PR TITLE
DBZ-3695 Community newsletter 01/2021

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-3695

### CODE REVIEW

**Review by gunnarmorling — APPROVED** [review]
Hey @ani-sha, excellent work with this newsletter edition! I've pushed one more commit with some changes and additions that came to my mind. Could you take a look at those and let me know if you have any concerns or doubts here? Besides that, I saw there's a few sentences directly copied from the last newsletter edition; Could you look into finding some alternative expressions for these? I know it can be challenging but it's always better to come up with different ways of saying things. Thanks!

**Comment by ani-sha** [comment]
@gunnarmorling Did a bit of rewording. Let me know if I can improve anything. Thanks.

File: _posts/2021-07-07-debezium-newsletter-01-2021.adoc [hunk 1]
```diff
@@ -142,6 +146,8 @@ Below is a list of open issues that are currently labeled with `easy-starter` if
 * Implement window function in MySQL parser (https://issues.redhat.com/browse/DBZ-3576[DBZ-3576])
 * Standardize "snapshot.fetch.size default" values across connectors (https://issues.redhat.com/browse/DBZ-3694[DBZ-3694])
 
+If you are new to open source, please check out our https://github.com/debezium/debezium.github.io/blob/develop/CONTRIBUTING.md[contributing guidelines] to get started!
```
Comment (gunnarmorling) [inline _posts/2021-07-07-debezium-newsletter-01-2021.adoc:149]: Nice!

**Comment by gunnarmorling** [comment]
And applied! Thank you so much, @ani-sha! Will share it via @debezium on Twitter once it's live.

**Pull request closed without merging**